internal/commands/app/create_inputs_test.go

```diff
@@ -346,3 +475,29 @@ func TestAppCreateInputsResolveDataLake(t *testing.T) {
 		assert.Equal(t, "123", expectedGroupID)
 	})
 }
+
+func TestFindDefaultPath(t *testing.T) {
+	t.Run("should return new incremented directory if provided directory already exists", func(t *testing.T) {
```

Comment:
I think its worth adding test cases here that we return `name-1` as a default path, and something like `name-9`.  we could probably convert this test code to a test table to reuse the test implementation